covid_graphs/covid_graphs/country_graph.py

```diff
@@ -59,19 +65,34 @@ def adjust_xlabel(date: datetime.date):
 
         colors = ["SteelBlue", "Purple", "Green", "Orange", "Gray"][: len(self.curves)]
 
-        # Traces
-        traces = []
+        traces, shapes = [], []
+        maximal_y = max(self.cropped_cumulative_active)
         for color, curve in zip(colors, self.curves):
+            xs, ys = curve.generate_trace(self.display_at_least_until)
+            idx_max = ys.argmax()
+            maximal_y = max(maximal_y, ys.max())
```

Comment:
I think this is a bug, `maximal_y` gets redefined as we iterate through curves. I suggest to rename on L69 to `max_cumulative_active`.

Reply:
Ah, sure, nevermind :)